### PR TITLE
mowing redo

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

stupid java and their nonhashable arrays

### CODE REVIEW

File: solutions/usaco-593.mdx [hunk 1]
```diff
@@ -7,138 +7,230 @@ author: Óscar Garries, Ryan Chou
...
+for dir_, num_steps in steps:
+	# Get the direction Farmer John is moving.
+	if dir_ == 'N':
+		delta = 0, 1
+	elif dir_ == 'W':
+		delta = -1, 0
+	elif dir_ == 'E':
+		delta = 1, 0
+	elif dir_ == 'S':
+		delta = 0, -1
```
Comment (ryanchou-dev) [inline solutions/usaco-593.mdx:217]: perhaps make delta more specific: `delta = [0, 1]`, ...

**Review by ryanchou-dev — APPROVED** [review]

File: solutions/usaco-593.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-593.mdx:217]: wdym by this?

Comment (ryanchou-dev) [reply solutions/usaco-593.mdx:217]: like write out the list brackets

this is totally optinoal

Comment (SansPapyrus683) [reply solutions/usaco-593.mdx:217]: rn `delta` isn't a list, it's a tuple
don't see any particular advantage to making it a list?

**Comment by ryanchou-dev** [comment]
oopsie, yeah go ahead, mb

**Review by TruaShamu — APPROVED** [review]

**Pull request merged**